Modules/_testlimitedcapi/long.c

```diff
@@ -671,7 +672,7 @@ pylong_aslonglongandoverflow(PyObject *module, PyObject *arg)
     int overflow = UNINITIALIZED_INT;
     long long value = PyLong_AsLongLongAndOverflow(arg, &overflow);
     if (value == -1 && PyErr_Occurred()) {
-        assert(overflow == -1);
+        assert(overflow == -1 || overflow == 0);
```

Comment:
Just same remark as above:
```suggestion
        // overflow can be 0 if a seperate exception occurred
        assert(overflow == -1 || overflow == 0);
```